CMakeLists.txt

```diff
@@ -74,6 +74,16 @@ ELSE()
     target_link_libraries(Hyprland xcb)
 ENDIF(NO_XWAYLAND MATCHES true)
 
+IF(USES_SYSTEMD MATCHES true)
```

Comment:
okay but it still doesnt automatically find it.

Reply:
so we want to set `USES_SYSTEMD` unconditionally _if_ `SYSTEMD_FOUND`?

Reply:
oh! `cmake` stuff is un-related to `meson`, of course.